debian/control

```diff
@@ -1,7 +1,7 @@
 Source: pistache
 Section: libdevel
 Priority: optional
-Maintainer: Dennis Jenkins <dennis.jenkins.75@gmail.com>
+Maintainer: Kip Warner <kip@thevertigo.com>
```

Comment:
Just a heads up @dennisjenkins75, this is the package maintainer's contact info, not upstream. In the Debian world they generally assume they're different.